Misha/OB-365_erat_sieve/utest_sieve_of_erat.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from main import sieve_of_eratosthenes
+
+
+class EratSieveTest(unittest.TestCase):
+
+    def test_erat_sieve_with_right_parameter(self):
+        primes = sieve_of_eratosthenes(6)
+        expected_result = [2, 3, 5]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_border_parameter(self):
+        primes = sieve_of_eratosthenes(5)
+        expected_result = [2, 3, 5]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_minimal_parameter(self):
+        primes = sieve_of_eratosthenes(2)
+        expected_result = [2]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_non_type_object(self):
+        with self.assertRaises(TypeError) as context:
+            self.assertRaises(TypeError, sieve_of_eratosthenes(None))
+        the_exception = str(context.exception)
```

Comment:
Change the_exception to exception_message

Reply:
corrected